onnxruntime/core/providers/coreml/builders/impl/clip_op_builder.cc

```diff
@@ -200,7 +198,9 @@ Status ClipOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
 bool ClipOpBuilder::IsOpSupportedImpl(const Node& node, const OpBuilderInputParams& input_params,
                                       const logging::Logger& logger) const {
   float min, max;
-  return GetClipMinMax(input_params.graph_viewer, node, min, max, logger);
+  bool ret = GetClipMinMax(input_params.graph_viewer, node, min, max, logger);
+  // what does it mean if min == max?
+  return ret && (min != max);
```

Comment:
Is that a real concern? I would expect it's a meaningless model if it clips to a single value.

Reply:
Yes, I believe it's meaningless for a constant clip.
But this model has such a node.
https://github.com/onnx/models/tree/main/Computer_Vision/fasterrcnn_resnet50_fpn_v2_Opset16_torchvision